src/components/Filters/Filters.types.ts

```diff
@@ -24,6 +24,7 @@ interface ComponentProps {
   maxDate?: Date;
   units?: string;
   placeholder?: string | DateRangePlaceholderProps;
+  validate?: (target: HTMLInputElement) => string | boolean;
```

Comment:
Pls, what is the use case for return of `string` by validation function? I would expect input value in, true or false out